src/passkit.cs

```diff
@@ -1171,27 +1213,29 @@ interface PKLabeledValue
 
 	[Mac (11,0)]
 	[Watch (4,3), iOS (11,3)]
-	[BaseType (typeof (NSObject))]
+	[BaseType (typeof (PKStoredValuePassProperties))]
 	[DisableDefaultCtor]
 	interface PKTransitPassProperties {
 
-		[Static]
-		[Export ("passPropertiesForPass:")]
-		[return: NullAllowed]
-		PKTransitPassProperties GetPassProperties (PKPass pass);
```

Comment:
that's a breaking change to remove this API

Reply:
When I see diffs like `-+ (nullable instancetype)passPropertiesForPass:(PKPass *)pass;` for something that leads to a breaking change, should I just ignore it then?